### ISSUE TITLE
`NotFound` when publishing package

### ISSUE THREAD

**Issue by BenStigsen** [blade-lang/nyssa#2]
I have created a publisher account and I've run `nyssa init`, which has left me with the following:
```json
{
  "name": "miniroute", 
  "version": "1.0.0", 
  "description": "Tiny and simple HTTP router.", 
  "homepage": "https://github.com/BenStigsen/miniroute", 
  "tags": [
    "http", 
    "router"
  ], 
  "author": "Benjamin Stigsen", 
  "license": "ISC", 
  "sources": [
    "https://nyssa.bladelang.com"
  ], 
  "deps": {}
}
```

If I run `nyssa info` I correctly get:
```yaml
PACKAGE INFORMATION
~~~~~~~~~~~~~~~~~~~
 • Name: miniroute
 • Version: 1.0.0
 • Description: Tiny and simple HTTP router.
 • Homepage: https://github.com/BenStigsen/miniroute
 • Author: Benjamin Stigsen
 • License: ISC
 • Tags: http, router

DEPENDENCIES
~~~~~~~~~~~~
  None

SOURCES
~~~~~~~
 • https://nyssa.bladelang.com
```

However, when I do any of the following:
```
nyssa publish
nyssa publish -r .
nyssa publish -r https://github.com/BenStigsen/miniroute
nyssa publish -r https://nyssa.bladelang.com/
```
It results in:
```
Checking for valid publisher account
Checking for valid Nyssa package
Packaging miniroute@1.0.0...
NotFound -> no such file or directory
```

I have reinstalled Nyssa and tried cleaning the cache.

**Comment by mcfriend99** [comment]
Please share a link to the package.

**Comment by mcfriend99** [comment]
It's a bug in the `zip` module of Blade. I'm raising a ticket to fix it now. In the meantime, in your Blade installation path, modify the path `libs/zip.b` and change line 587 from:

```
if npath.starts_with('/')
```

To

```
if npath.starts_with(os.path_separator)
```

You should be able to publish after that.

**Comment by BenStigsen** [comment]
Yes this made me able to publish! https://nyssa.bladelang.com/view/miniroute
One thing does surprise me though, how come it does not show `1.0.1` by default, but instead shows version `1.0.0`?

**Comment by mcfriend99** [comment]
That's a bug. I'll fix now.

**Comment by BenStigsen** [comment]
I was just wondering if it was a design choice.
I don't mean to stress you or take your focus away from other tasks 😄

**Comment by mcfriend99** [comment]
Fixed in `v0.1.7`
